site/docs/recipe_hello_quarks.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Recipe 1. Hello Quarks!
+---
+
+Quarks' pure Java implementation is a powerful feature which allows it to be run on the majority of JVM-compatible systems. It also has the added benefit of enabling the developer to develop applications entirely within the Eclipse and Intellij ecosystems. For the purposes of this (and future) recipes, it will be assumed that the developer is using Eclipse. To begin the Hello World recipe, create a new project and import the necessary libraries as outlined in the [getting started guide](quarks-getting-started). Next, write the following template application:
```

Comment:
I put a comment in QUARKS-16

Looks good though maybe we should not have all recipes directly in the `docs` directory, but at least a `recipes` or `cookbook` folder and maybe a sub-directory under that for each recipe?

Image if we had 100 recipes ...

https://issues.apache.org/jira/browse/QUARKS-16?focusedCommentId=15205345&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15205345